website/docs/releases/2024/v2024.2.md

```diff
@@ -0,0 +1,10180 @@
+
+    -   `authentik_outpost_flow_timing_get` -> `authentik_outpost_flow_timing_get_seconds`
+    -   `authentik_outpost_flow_timing_post` -> `authentik_outpost_flow_timing_post_seconds`
+    -   `authentik_outpost_ldap_requests` -> `authentik_outpost_ldap_request_duration_seconds`
+    -   `authentik_outpost_ldap_requests_rejected` -> `authentik_outpost_ldap_requests_rejected_total`
+    -   `authentik_outpost_proxy_requests` -> `authentik_outpost_proxy_request_duration_seconds`
+    -   `authentik_outpost_proxy_upstream_time` -> `authentik_outpost_proxy_upstream_response_duration_seconds`
+    -   `authentik_outpost_radius_requests` -> `authentik_outpost_radius_request_duration_seconds`
+    -   `authentik_outpost_radius_requests_rejected` -> `authentik_outpost_radius_requests_rejected_total`
+    -   `authentik_main_requests` -> `authentik_main_request_duration_seconds`
+
+-   The shorthand parameter for `--stage`, `-s` for the `ak test_email` command has been changed to `-S`
+
+## New features
+
+-   **New provider: Remote Access Control** <span class="badge badge--primary">Enterprise</span>
+
+    The Remote Access Control provider allows you to remotely connect to remote machines over RDP, SSH and VNC through authentik. As such, you can use the same policy engine and customization options that are possible with other providers using the same user and admin interface.
+
+-   **Audit log** <span class="badge badge--primary">Enterprise</span>
+
+    authentik instances which have a valid enterprise license installed will log changes made to models including which fields were changed with previous and new values of the fields. The values are censored if they are sensitive (for example a password hash), however a hash of the changed value will still be logged.
+
+-   **Session location and network binding**
+
+    Sessions for any users can now be bound to a specific geolocation (Continent, Country, City) or network (Autonomous System, subnet, IP address). If the session is accessed from a different location/network it was initially created from, the session will be terminated.
+
+    Configuration steps are available [here](../../flow/stages/user_login/index.md#network-bindinggeoip-binding).
+
+-   **S3 file storage**
+
+    Media files can now be stored on S3. Follow the [setup guide](../../installation/storage-s3.md) to get started.
+
+-   **_Pretend user exists_ option for Identification stage**
+
+    Previously the identification stage would only continue if a user matching the user identifier exists. While this was the intended functionality, this release adds an option to continue to the next stage even if no matching user was found. "Pretend" users cannot authenticate nor receive emails, and don't exist in the database.
+
+    **This feature is enabled by default.**
+
+-   **Tenancy** <span class="badge badge--primary">Enterprise</span>
+
+    :::warning
+    This feature is in early preview. Use at your own risk.
```

Comment:
Perhaps less scary wording, like "This feature is in early preview. We recommend backing up your instance before using." ??

Reply:
I like it scary. We don't have much experience with it, and until we do, I wouldn't recommend people use it.